[2025-03-17T15:16:18.924+0600] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-17T15:16:18.947+0600] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_airflow_rest_api_dag.load_weather_task scheduled__2025-03-10T18:00:00+00:00 [queued]>
[2025-03-17T15:16:18.963+0600] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_airflow_rest_api_dag.load_weather_task scheduled__2025-03-10T18:00:00+00:00 [queued]>
[2025-03-17T15:16:18.964+0600] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-03-17T15:16:18.984+0600] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): load_weather_task> on 2025-03-10 18:00:00+00:00
[2025-03-17T15:16:19.000+0600] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'etl_airflow_rest_api_dag', 'load_weather_task', 'scheduled__2025-03-10T18:00:00+00:00', '--job-id', '60', '--raw', '--subdir', 'DAGS_FOLDER/etl_airflow_rest_api_open_weather_data.py', '--cfg-path', '/tmp/tmp1t1k0pyg']
[2025-03-17T15:16:19.003+0600] {standard_task_runner.py:105} INFO - Job 60: Subtask load_weather_task
[2025-03-17T15:16:19.008+0600] {logging_mixin.py:190} WARNING - /home/shamim/airflow_venv/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=21112) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-03-17T15:16:19.009+0600] {standard_task_runner.py:72} INFO - Started process 21119 to run task
[2025-03-17T15:16:19.053+0600] {task_command.py:467} INFO - Running <TaskInstance: etl_airflow_rest_api_dag.load_weather_task scheduled__2025-03-10T18:00:00+00:00 [running]> on host shamim-X541UV
[2025-03-17T15:16:19.124+0600] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='etl_airflow_rest_api_dag' AIRFLOW_CTX_TASK_ID='load_weather_task' AIRFLOW_CTX_EXECUTION_DATE='2025-03-10T18:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-10T18:00:00+00:00'
[2025-03-17T15:16:19.126+0600] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-03-17T15:16:19.127+0600] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-03-17T15:16:19.127+0600] {logging_mixin.py:190} INFO - Current task name:load_weather_task state:running start_date:2025-03-17 09:16:18.948374+00:00
[2025-03-17T15:16:19.127+0600] {logging_mixin.py:190} INFO - Dag name:etl_airflow_rest_api_dag and current dag run status:running
[2025-03-17T15:16:19.128+0600] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-17T15:16:19.141+0600] {base.py:84} INFO - Retrieving connection 'postgres'
[2025-03-17T15:16:19.159+0600] {sql.py:624} INFO - Running statement: 
    INSERT INTO weather_data_api (city, temperature, humidity)
    VALUES(%s, %s, %s);
    , parameters: ('Dhaka', 31.99, 29)
[2025-03-17T15:16:19.161+0600] {sql.py:633} INFO - Rows affected: 1
[2025-03-17T15:16:19.162+0600] {python.py:240} INFO - Done. Returned value was: None
[2025-03-17T15:16:19.171+0600] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-17T15:16:19.171+0600] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=etl_airflow_rest_api_dag, task_id=load_weather_task, run_id=scheduled__2025-03-10T18:00:00+00:00, execution_date=20250310T180000, start_date=20250317T091618, end_date=20250317T091619
[2025-03-17T15:16:19.379+0600] {logging_mixin.py:190} INFO - Task instance in success state
[2025-03-17T15:16:19.380+0600] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-03-17T15:16:19.381+0600] {logging_mixin.py:190} INFO - Dag name:etl_airflow_rest_api_dag queued_at:2025-03-17 09:15:34.281367+00:00
[2025-03-17T15:16:19.382+0600] {logging_mixin.py:190} INFO - Task hostname:shamim-X541UV operator:PythonOperator
[2025-03-17T15:16:19.426+0600] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-17T15:16:19.440+0600] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-17T15:16:19.441+0600] {local_task_job_runner.py:245} INFO - ::endgroup::
